scanpy/external/tl/_trimap.py

```diff
@@ -0,0 +1,148 @@
+from ... import logging as logg
+import numpy as np
+import scipy.sparse as scp
+
+
+def trimap(
+    adata,
+    n_dims=2,
```

Comment:
Please add a `*,` after `adata,` or `n_dims: int = 2,`.

This way we’re free to reorganize subsequent parameters without it being a breaking change.